.github/workflows/after-merging.yml

```diff
@@ -0,0 +1,51 @@
+# Workflow to run accessibility tests after merging using Cypress 
```

Comment:
For the new workflows (cypress, attaching to Trello and merging dependabot PRs) is it worth adding something to the readme to explain what they are and the need to refer to these files and add the appropriate secrets to your repo/update dashboard name?

Reply:
Definitely! Thanks Kate, will do